echopype/echodata/echodata.py

```diff
@@ -141,7 +141,7 @@ def _load_group(filepath, group=None):
 
         return xr.open_dataset(filepath, group=group, engine=XARRAY_ENGINE_MAP[suffix])
 
-    def to_netcdf(self, **kwargs):
+    def to_netcdf(self, save_path, **kwargs):
```

Comment:
This seem to require user now to need `save_path`. Is this intended. We have a warning right now if there is no `save_path`.

Reply:
> need is to initialize save_path=None here?

Yes.